### PR TITLE
k8s: return ports in process list

### PR DESCRIPTION
**Pull request by beastawakens**

### CODE REVIEW

File: provider/k8s/process.go [hunk 1]
```diff
@@ -569,14 +569,18 @@ func (p *Provider) processFromPod(pd ac.Pod) (*structs.Process, error) {
 
 	ports := []string{}
 	for _, p := range cs[0].Ports {
-		ports = append(ports, fmt.Sprintf("%d:%d", p.HostPort, p.ContainerPort))
+		if p.HostPort == 0 {
```
Comment (ddollar) [inline provider/k8s/process.go:572]: Make sure to add a test for this case

File: provider/k8s/process_test.go [hunk 2]
```diff
@@ -6,7 +6,6 @@ import (
 
 	"github.com/convox/convox/pkg/structs"
 	"github.com/convox/convox/provider/k8s"
-
```
Comment (ddollar) [inline provider/k8s/process_test.go:9]: This likely violates gofmt

File: provider/k8s/process.go (hunk 1 above)
Comment (beastawakens) [reply provider/k8s/process.go:572]: Yep, there is, the 3rd one in the list!

**Comment by beastawakens** [comment]
> * make sure to run `gofmt` on all files you edit.. probably want to do this automatically on save
> * structure your commits to be fairly atomic with descriptive commit messages, avoid "WIP"

- I swore I had code set up to do that... 🤔 I'll check it.
- Yeah, that crept in from rebasing from the previous, now defunct, branch 😛

**Comment by beastawakens** [comment]
@ddollar All sorted now!

**Pull request merged**